ccd-definition/AuthorisationCaseField/solicitor.json

```diff
@@ -236,5 +236,26 @@
     "CaseFieldID": "responseIntention",
     "UserRole": "caseworker-civil-solicitor",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseFieldID": "respondentResponseType",
+    "UserRole": "caseworker-civil-solicitor",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseFieldID": "respondentResponseDocument",
+    "UserRole": "caseworker-civil-solicitor",
+    "CRUD": "CRU"
+  },
```

Comment:
does this naming help? It needs to be be respondent1 OR just responseType

Reply:
As claimant will also have a response after the defendant has responded, so `respondentResponseType` make more sense than just `responseType`